src/jit/gentree.h

```diff
@@ -598,9 +598,14 @@ struct GenTree
         return isIndir() || isLclField();
     }
 
-    bool isContainedMemoryOp() const
+    bool isUsedFromMemory() const
     {
-        return (isContained() && isMemoryOp()) || isContainedLclVar() || isContainedSpillTemp();
+        return ((isContained() && (isMemoryOp() || isContainedLclVar())) || isUsedFromSpillTemp());
```

Comment:
Did this intentionally change from `(isContained() && isMemoryOp()) || isContainedLclVar())` to `(isContained() && (isMemoryOp() || isContainedLclVar()))`?